fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -129,8 +136,10 @@ The general parameters are defined in the `default` key and can be:
 | `padding` | The content of the framed screenshot will be resized to match the specified `padding` around all edges. The vertical padding is also applied between the text and the top or bottom (depending on `title_below_image`) of the device frame. <p> There are 3 different options of specyfying the padding: <p> 1. Default: An integer value that defines both horizontal and vertical padding in pixels. <br> 2. A string that defines (different) padding values in pixels for horizontal and vertical padding. The syntax is `"<horizontal>x<vertical>"`, e.g. `"30x60"`. <br> 3. A string that defines (different) padding values in percentage for horizontal and vertical padding. The syntax is `"<horizontal>%x<vertical>%"`, e.g. `"5%x10%"`. <br> **Note:** The percentage is calculated from the smallest image dimension (height or width). <p> A combination of option 2 and 3 is possible, e.g. `"5%x40"`. | `50` |
 | `interline_spacing` | Specifies whether _frameit_ should add or subtract this many pixels between the individual lines of text. This only applies to a multi-line `title` and/or `keyword` to expand or squash together the individual lines of text. | `0` |
 | `font_scale_factor` | Specifies whether _frameit_ should increase or decrease the font size of the text. | `0.1` |
-| `frame` | Overrides the color of the frame to be used. (Valid values are `BLACK, `WHITE`, `GOLD` and `ROSE_GOLD`) | NA |
+| `frame` | Overrides the color of the frame to be used. (Valid values are `BLACK`, `WHITE`, `GOLD` and `ROSE_GOLD`) | NA |
```

Comment:
Is this still limited to these options or can the new colors be used here as well?

Reply:
Yes, I forgot to update this part of the documentation. Shall I add the list of all supported colors, maybe separating iOS and Android colors?

Reply:
That is actually a lot more complicated now, because when this was first written all colors were available for all devices. This obviously hasn't been true for quite some time now.

But that shouldn't be your problem, so we can decide now if it's better to just add them to the list or leave them off - but create a new issue so this gets fixed and documented better (simplest solution I can think of right now is for this just linking to a "ABout device colors" headline that then has a separated list for iOS and Android, or maybe even a table which devices are available in what color.

Reply:
OK, I have created a feature request. #15089